tgui/packages/tgui/interfaces/Fabrication/MaterialIcon.tsx

```diff
@@ -84,7 +85,7 @@ export const MaterialIcon = (props: MaterialIconProps) => {
 
   while (
     icons[activeIdx + 1] &&
-    icons[activeIdx + 1][0] <= (amount ?? 200_000) / 2_000
+    icons[activeIdx + 1][0] <= (amount ?? 200_000) / 2_000 // todo: how does this work exactly? change to use the define
```

Comment:
this is why it's drafted lol
i'm not sure if passing the define through `Material` is the cleanest solution and i'm not sure how does the equation here work. please advise